src/features/tabsWithNumbers.ts

```diff
@@ -84,9 +85,9 @@ export default () => {
                     }
                 }
 
-                const tabIndex = (getExtensionSetting('showTabNumbers.reversedMode') ? [...recentFileStack].reverse() : recentFileStack).findIndex(
-                    elemUri => elemUri.toString() === uri.toString(),
-                )
+                const tabIndex = (getExtensionSetting('showTabNumbers.reversedMode') ? [...recentFileStack].reverse() : recentFileStack)
+                    .filter(({ scheme }) => scheme === 'file')
```

Comment:
Why do we need filter `recentFileStack`?

1. I don't see a reason for this change
2. We can just not add uri to this array if it doesn't satisfy some requirement, which is also unclear

Reply:
I agree filtering `recentFileStack` was bad idea, but I the purpose of this change is removing file decoration from the `search-editor`, so I decided to skip search-editors from adding to `recentFIlestack` at all.